main/src/com/google/refine/templating/Parser.java

```diff
@@ -81,7 +81,7 @@ static public Template parse(String s) throws ParsingException {
                     continue;
                 }
             } else if (c == '{' && c2 == '{') {
-                int closeBrace = s.indexOf('}', current + 2);
+                int closeBrace = s.indexOf("}}", current + 2);
                 if (closeBrace > current + 1 && closeBrace < s.length() - 1 && s.charAt(closeBrace + 1) == '}') {
```

Comment:
It looks like the last condition is no longer necessary:
```suggestion
                if (closeBrace > current + 1 && closeBrace < s.length() - 1) {
```

Reply:
Will the second one becomes not necessary too? As if there's no "}}", the closeBrace will have value -1. Otherwise, the closeBrace will always be s.length() - 1

Reply:
Good point!